content/challenges/easy/5 - Hello Tensure/func.js

```diff
@@ -11,5 +11,13 @@
  * @returns {string}
  */
 module.exports = (num) => {
-    // your code here
+    if (((num % 3) + (num % 5)) === 0) {
+        return "Hello Tensure";
+    } if (num % 3 === 0) {
+        return "Hello";
+    } if (num % 5 === 0) {
+        return "Tensure";
+    } else {
+        return "Goodbye Tensure";
```

Comment:
Nice! Yup you have a default case here. there is no more logic to check with the instructions you were given!